### ISSUE TITLE
macOS move application file from home directory to Application Folder

### ISSUE THREAD

**Issue by akasolace** [akasolace/HO#5]
2 possibilities:
 - find a way to set chdir on os x startup script (does not look feasible)
 - move from relative to absolute path on mac os only (possible lot of work and very ennoying to do)

**Comment by akasolace** [comment]
@pierpaolocira  I was cleaning old issues and I found this one which is link or even similar to #372 
So in case, I also assigned it to you for the day you will be able to tackle those ...

**Comment by akasolace** [comment]
@pierpaolocira the way I see it this is a duplicate of #372 so I am closing this one. If you disagree you can reopen it ... 
cheers

**Comment by pierpaolocira** [comment]
I do confirm... it's the same, although the final location is different. Here it is proposed to put it into the Application folder, but it doesn't sound as a good idea because it might be lost/overwritten easily (and MacOs uses another default location for such kind of information, which is what I used in 372, i.e. "application support"). It's a pity I have no time for proper testing, because it's ready since a long time... but I will feel confident to replease only after proper testing from me (and then from some voulunteer)...

**Comment by akasolace** [comment]
@pierpaolocira I do agree we need proper testing first.  That would be a very nice improvement so really looking forward to it ...
